README.md

```diff
@@ -1,9 +1,15 @@
-# Mazzaroth-xdr
+# Mazzaroth-XDR
+
 [![CircleCI](https://circleci.com/gh/kochavalabs/mazzaroth-xdr.svg?style=svg)](https://circleci.com/gh/kochavalabs/mazzaroth-xdr)
+**[Self Link](https://github.com/kochavalabs/mazzaroth-xdr)**
```

Comment:
What is this being added for?

Should it be moved down at least so that its not right next to the CirlceCI status badge?

Reply:
It's nice to have a link to the doc for when we import it to other places, like our docs site.

It should be moved down a line, one second.